code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -155,6 +157,7 @@
 /obj/structure/cult/functional/altar/Initialize(mapload)
 	. = ..()
 	icon_state = SSticker.cultdat?.altar_icon_state
+	cooldowntime = CULT_STRUCTURE_COOLDOWN
```

Comment:
world.time + CULT_STRUCTURE_COOLDOWN
right now it'll just set the cooldown to 600 deciseconds, which won't do anything if the round is past that point